back/spec/services/json_forms_service_spec.rb

```diff
@@ -348,7 +348,7 @@
           'en' => '<img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" />'
         }
         field = create :custom_field, :for_custom_form, input_type: 'text', description_multiloc: description_multiloc
-        expect_any_instance_of(TextImageService).to(
+        allow_any_instance_of(TextImageService).to(
```

Comment:
Why was this changed? Don't we want to make sure that the text image service is used to process images in HTML?

Reply:
It was changed because the message is sent multiple times now, once per locale, which was not the case before.

Reply:
Do you mean that the tests does not verify that it is called? That is a side effect. How can we solve that?

Reply:
I take that back. The test verifies that it is called, because it has an expectation on the return value of the spec. So all is good.

Reply:
Yes indeed, the test verifies this implicitly, but I thought it could be nice to also have an explicit check. I had hoped it would still work for any number of times (at least once) the stubbed method is called. Not blocking for me in any case.